input-output-hk/daedalus
# [DDW-853] Update electron

This PR updated electron and related package versions.
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-853)

## Todos
- [x]  Update electron and its related packages to latest stable versions.
- [x]  Check `yarn.lock` to see if bad entires were added
- [x]  Check terminal to see if everything was installed properly
- [x]  Make sure that all `yarn ...` scripts work well
- [x] Fix "Blank screen fix" issues on macOS Mojave/Catalina
- [x] Fix "Wallet import" state folder selection
- [ ] ~Persist "Blank screen fix" active status after Daedalus restart~ - To be resolved in [DDW-155](https://jira.iohk.io/browse/DDW-155)

## Test Checklist
- [x]  Make sure all file open/save dialogs work well.
       * Go to `Wallet File Import` screen and check if file import works well.
       * Go to `Staking Rewards` screen and check if csv export works well.
       * Go to `Wallet Receive` screen and check if pdf download works well.
       * Go to `Paper Wallet Certificate` => `Instructions` dialog and check if print works well.
       * Open `Export Wallet to File` dialog and check if export works well.
       * Check if `Download Logs` works well.
- [x]  Make sure logs are printed out well on the console.
- [x]  Make sure all the user settings are saved well on settings.

## Review Checklist

### Basics


### Code Quality

### Testing

### After Review
